app/src/main/res/layout/login_activity.xml

```diff
@@ -0,0 +1,30 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
This makes it better when generating layout bindings. When using layout binding, it won't matter what is the naming style. Furthermore, if we follow the project style of package by feature. login_* should be followed since activity_* is package by layer